docs/identityManager.md

```diff
@@ -17,6 +17,28 @@ The IdentityManager should be able to perform the following actions:
 ## Design
 The IdentityManager contract has any number of owners and one recovery key for each proxy. It also has rate limits for each caller on some of the functions. Proxies can be created, transfered to and from the IdentityManager. Owners can be added and removed, the recovery key can be changed.
 
+### Values of constants
+There are three constants that needs to be set when creating this contract. All of these constants are specified in seconds.
+```
+_userTimeLock - Time before new owner added by recovery can control proxy
+_adminTimeLock - Time before new owner can add/remove owners
+_adminRate - Time period used for rate limiting a given key for admin functionality
+```
+
+We set these values to the following:
+
+|Constant|Value|
+| --|--|
+|_userTimeLock|3600 (1 hour)|
+|_adminTimeLock|129600 (1.5 days)|
+|_adminRate|1200 (20 minutes)|
```

Comment:
I have nothing against these values per se but what worries me is that we're adding a lot of complexity right out of the gate that will carry through to the mobile app and our whole stack.

We have not done any work on how we think the UX should work around these timeouts (i.e. having to wait 1 hour after a recovery before you can send transactions), also it's not clear how these will work together with the simple ID stuff. Also it's going to be a lot of UX work until a user will actually be able to react to a takeover attempt in any meaningful way.

Not sure what the best thing to do is. I feel like if we don't have the UX to be able to handle a 1 hour timeout for instance we should just be setting the timeout to zero.

Reply:
Yeah, I agree that this is not something that is easy to get right. I asked about this on slack but didn't get any response. However I think we need to make a decision now. This might be something that is not possible to get right the first time around, so worst case we have to update the deployed contract.

Reply:
@oed I think those values are good. The only one I might increase is adminRate to 60 minutes as well

Don't worry about the UX issues for now. We'll deal with them.